packages/eslint-config-nori/index.js

```diff
@@ -214,6 +214,11 @@ module.exports = {
           {
             selector: ['parameter'],
             format: ['camelCase'],
+          },
+          {
+            selector: ['parameter'],
+            format: ['camelCase'],
+            modifiers: ['unused'],
```

Comment:
```suggestion
```

Tightening this rule to only allow unused parameters to start with an underscore introduces a new set of eslint issues - since we aren't following that rule across the board. I'm removing this bit